sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -191,7 +191,7 @@ def test_pca_vs_spca():
     assert_allclose(results_test_pca, results_test_spca)
 
 
-@pytest.mark.parametrize("SPCA", [SparsePCA, MiniBatchSparsePCA])
+@pytest.mark.parametrize("SPCA", (SparsePCA, MiniBatchSparsePCA))
```

Comment:
Let's not introduce unrelated minor changes like this.

```suggestion
@pytest.mark.parametrize("SPCA", [SparsePCA, MiniBatchSparsePCA])
```

Reply:
I see thanks.